src/test/java/com/qa/test/classTest1.java

```diff
@@ -6,6 +6,6 @@
 
 	@Test
 	public void firstTest() {
-		System.out.println("git learning project");
```

Comment:
please change

Reply:
Hi,
i have updated branch2 for changes asked for.
please check